TAC/Meetings/2022/2022-02-10/CCC_TAC_Minutes-2022-02-10.md

```diff
@@ -0,0 +1,182 @@
+
+MB noted that hard-to-follow processes put people at a disadvantage. He requested clarification why CoC escalations go through the TAC vs. the governing board. DT noted that CCC attempts to follow the mantra of "Minimum Viable Governance." He noted that there is a requirement for projects to submit their CoC, but that he would welcome a PR to add an escalation process requirement.
+
+SW asked for next steps. DM also suggested a PR adding a requirement for projects to publish an escalation path. AB recommended the escalation path be explicitly defined, and the people in the escalation path should have been trained to go through the CoC process. DT recommended also providing recommendations broadly across the CCC.
+
+DM asked if a specific escalation process is required. AB noted there may be a need to have a specific set of individuals who are more knowledgeable of CCC dynamics who are centered in the process.
+
+BW reviewed the process of escalation at the Linux Foundation when a report is escalated beyond a project. He noted that the LF almost always lacks specific context, and will involve the project community for context anyhow. In the interest of ensuring a functional process, he recommended the project specify its own escalation path with the Linux Foundation as fallback.
+
+**ACTION:** Stephen Walli to work with Dan Middleton, Aeva Black, and others to develop an escalation process, to be presented to TAC and governing board.
+
+**ACTION:** Dan Middleton to open a pull request adding a defined escalation process to project requirements.
+
+### Process improvement suggestions
+
+BW reviewed a number of process improvement suggestions. Dave noted that process improvements shouldn't require a TAC vote. Consensus was reached that if there are improvements that will enhance the efficiency of the TAC, that they should be made transparency. 
+
+Jethro Beekman (JB) noted these are public meetings and everyone is welcome to attend, but there is still some distance between open meetings and recordings being broadly available to the whole internet. He noted this could create hesitance that would prevent people from participating in meetings. SW provided historical context, noting meeting recordings are the backup to the minutes. He reviewed historical concerns about posting recordings. DT noted there are two questions. Are they stored on YouTube? And are they private, unlisted, or public?
```

Comment:
```suggestion
Jethro Beekman (JB) noted these are public meetings and everyone is welcome to attend, but there is still some distance between open meetings and recordings being broadly available to the whole Internet. He noted this could create hesitance that would prevent people from participating in meetings. SW provided historical context, noting meeting recordings are the backup to the minutes. He reviewed historical concerns about posting recordings. DT noted there are two questions. Are they stored on YouTube? And are they private, unlisted, or public?
```